Remotion/Validation/Globalization/LocalizedValidationMessageFactory.cs

```diff
@@ -184,6 +193,30 @@ private ResourceIdentifier GetResourceIdentifierForLengthValidator (LengthValida
       return ResourceIdentifier.ValueMustMatchMinAndMaxLengthValidationMessage;
     }
```

Comment:
Please simplify the GetResourceIdentifierForLengthValidator implementation. I think we can simplify to use just `return ResourceIdentifier.ValueMustMatchMinAndMaxLengthValidationMessage;`. the Max is no longer optional, Min=0 and Min=Max will be forbidden in the follow-up tasks and until that time, it's just a grammar speciliaty which we can safely ignore.